index.js

```diff
@@ -227,11 +228,13 @@ PersonaClient.prototype.validateToken = function (token, scope, overridingScope,
  * @param {Object} request - HTTP request object.
```

Comment:
Might be worth adding a note here that if you want to validate against a scope, provide it as `request.params.scope` in case people read this over release notes?